paramak/parametric_components/tf_coil_casing.py

```diff
@@ -100,5 +141,13 @@ def create_solid(self):
             self.azimuth_placement_angle
         solid = union_solid(solid, self.leg_shape)
         solid = cut_solid(solid, self.magnet)
+
+        inner_bore_cutting_shape = ExtrudeStraightShape(
```

Comment:
@billingsley-john perhaps some comments could help understand what you did here (nice find btw)

Reply:
maybe even refer to the issue

Reply:
Sounds great, I shall add some!